locale/fi/LC_MESSAGES/django.po

```diff
@@ -3574,6 +3581,82 @@ msgstr "Ei osumia"
 msgid "Search for..."
 msgstr "Etsi..."
 
+#: templates/support_channels_en.html templates/support_channels_fi.html
+msgid "Documentation"
+msgstr "Dokumentaatio"
+
+#: templates/support_channels_en.html templates/support_channels_fi.html
+msgid "Aplus manual"
+msgstr "Aplus-käsikirja"
+
+#: templates/support_channels_en.html templates/support_channels_fi.html
+msgid "Aplus website"
+msgstr "Aplus-verkkosivusto"
+
+#: templates/support_channels_en.html templates/support_channels_fi.html
+msgid "Download course templates"
+msgstr "Lataa kurssimallit"
```

Comment:
I would say  that "course template" is "kurssipohja". "Malli" sounds like it is the only correct way to do it, e.g., "mallivastaus" is the correct solution. A template is not a (one and only correct) model solution.
Plural "course templates" "kurssipohjat".